controllers/index.js

```diff
@@ -23,6 +23,11 @@ exports.index = (req, res) => {
             const user = gitData[1];
             const prs = [];
 
+            if (user.data.type !== 'User') {
```

Comment:
For an organisation `user.data.type` won't exist (I think?) so do we need to check for that as well?

Reply:
Through my testing, it seems that `user.data.type` will be `'Organization'`.